eng/AzSdk.test.reference.targets

```diff
@@ -1,4 +1,5 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('test.props'))" />	
```

Comment:
Why is that needed we shouldn't be importing that test.props into our projects.

Reply:
Was trying to figure out why some tests were failing for no windows OS.